triage/subscribe-to-org-repos.js

```diff
@@ -0,0 +1,52 @@
+const axios = require('axios');
+
+// Replace with your GitHub token and organization name
+const GITHUB_TOKEN = 'token';
+const ORGANIZATION_NAME = 'pillarjs';
```

Comment:
It could be passed as command-line arguments using process.argv

```
const ORGANIZATIONS = process.argv.slice(2)

for (const arg of ORGANIZATIONS) {
...
}
```

Or simply add the three organizations.